src/dynamicTypes/eventHandlersService.ts

```diff
@@ -0,0 +1,21 @@
+import { EventHandler, ComponentsEventHandlers } from "../../scripts/types";
+import Constants from "../../scripts/constants";
+import eventHandlersJSON from "../../dist/eventHandlers.json";
+
+const eventHandlers = eventHandlersJSON as ComponentsEventHandlers;
+
+const removeModulePrefix = (componentName: string) =>
+  componentName.startsWith(Constants.EVENTS_MODULE_PREFIX)
+    ? componentName.slice(Constants.EVENTS_MODULE_PREFIX.length)
+    : componentName;
+
+const eventHandlersService = {
+  getComponentEventHandlers: (
+    componentName: string
+  ): EventHandler[] | undefined => {
+    const nameWithoutModule = removeModulePrefix(componentName);
```

Comment:
it's internal logic so it doesn't really matter, but why remove the prefix instead of making sure our list of events includes the full component name ?

Reply:
Yeah I'll do that.